src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSession.kt

```diff
@@ -1,35 +1,53 @@
 package de.bund.digitalservice.useid.identification
 
+import io.hypersistence.utils.hibernate.type.array.ListArrayType
+import org.hibernate.annotations.Type
+import org.hibernate.annotations.TypeDef
+import org.hibernate.annotations.TypeDefs
 import org.springframework.data.annotation.CreatedDate
-import org.springframework.data.annotation.Id
 import org.springframework.data.annotation.LastModifiedDate
-import org.springframework.data.relational.core.mapping.Column
-import org.springframework.data.relational.core.mapping.Table
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
 import java.time.LocalDateTime
 import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Table
 
-@Table("identification_session")
+@TypeDefs(
+    value = [
+        TypeDef(name = "list-array", typeClass = ListArrayType::class)
+    ]
+)
+@Table(name = "identification_session")
+@Entity
+@EntityListeners(AuditingEntityListener::class)
 data class IdentificationSession(
-    @Column("useid_session_id")
-    val useIdSessionId: UUID,
+    @Column(name = "useid_session_id")
+    var useIdSessionId: UUID? = null,
```

Comment:
Does anyone remember why we had to set this and the refreshAddress to null?

Reply:
Spring Data needs a default constructor with empty params to create the object and then set values on it. Providing default values is a way to accomplish that in Kotlin. There might be a better way though, don't know.